### PR TITLE
Oleksandr/branch

### PR DESCRIPTION
**Pull request by Oleksandr1823**
Сreated User feature

### CODE REVIEW

File: ProjectToTest/User.cs [hunk 1]
```diff
@@ -0,0 +1,21 @@
+﻿using System.Reflection;
```
Comment (frostysh) [inline ProjectToTest/User.cs:1]: I see a new class "User". I see.

**Review by frostysh — APPROVED** [review]
The new class "User" approved by mineself.

**Comment by olegkarman** [comment]
LG

**Review by olegkarman — APPROVED** [review]

File: ProjectToTest/User.cs (hunk 1 above)
Comment (Oleksandr1823) [reply ProjectToTest/User.cs:1]: thanks for the comment

File: ProjectToTest/User.cs [hunk 2]
```diff
@@ -0,0 +1,21 @@
...
+
+namespace ProjectToTest
+{
+    internal class User
+    {
+        public string FirstName { get; set; }
+        public string LastName { get; set; }
+        public int Age { get; set; }
+        public User(string firstName, string lastName, int age)
```
Comment (frostysh) [inline ProjectToTest/User.cs:10]: The question:
1) First we initialize properties via constructor.
2) Then we initialize properties via object initializer (the parenthesises `{}`).

Why? Why we need two times setup properties? Maybe I lost something???

File: ProjectToTest/User.cs (hunk 1 above)
Comment (frostysh) [inline ProjectToTest/User.cs:1]: Why we need System.Reflection, again??? It is hard to me to understand, well first I do not know what this class can do, but word 'reflection', looks to me without connection to the tasks of the excercise,

**Pull request merged**